SystemServices/SystemServices.h

```diff
@@ -209,6 +212,9 @@ namespace WPEFramework {
                 uint32_t setNetworkStandbyMode (const JsonObject& parameters, JsonObject& response);
                 uint32_t getNetworkStandbyMode (const JsonObject& parameters, JsonObject& response);
                 uint32_t getPowerStateIsManagedByDevice(const JsonObject& parameters, JsonObject& response);
+                uint32_t fwPendingReboot (const JsonObject& parameters, JsonObject& response);
+                uint32_t fwDelayReboot (const JsonObject& parameters, JsonObject& response);
+                uint32_t fwAutoReboot (const JsonObject& parameters, JsonObject& response);
```

Comment:
Change above methods to
setFirmwareAutoReboot
fireFirmwarePendingReboot
setFirmwareRebootDelay

Reply:
Changed.